### ISSUE TITLE
Formalize a "hospitality request" instead of a free-form message

### ISSUE THREAD

**Issue by rfay** [warmshowers/Warmshowers.org#310]
This concept was requested in https://www.warmshowers.org/content/host-requests-instead-messages

"It would be nice if there was a more structured way to request a stay with a host. Instead of a 'Send Message' button I'd like to see a 'Request a Stay' button. This would direct you to a form to select the estimated arrival date, time, and uncertainty, length of stay, and how many guests. Then there would be a message field to introduce yourself and write whatever you want."

Of course, Couchsurfing seems to have this pretty well nailed, so would be good to study their approach. 

This could also give us some good hosting statistics, which we sadly lack.

I think this could be done with a special-format message... Fields on the hosting request?

**Comment by kalpaitch** [comment]
I agree with formalising the request-a-stay process in such a way that gathers more hosting statistics.

However, I tend to agree with https://github.com/warmshowers/Warmshowers.org/issues/310#issuecomment-21142272. I have spoken to many travellers over the last year who prefer using WS precisely because the process isn't so formal as CS. Providing too many extra fields to fill in would I think go against this. Whenever I use CS I can never be bothered to go through the formal request process and I just send messages instead, I know others do this too.

Perhaps providing people with an example of the request information required would be a suitable middle ground?

**Comment by rfay** [comment]
We got the message loud and clear that it has to be simple. Everybody interested in feature priorities should spend a half hour looking at https://www.warmshowers.org/feature_poll and seeing the massive community input there.

However, it's my opinion that this feature can make things easier from both the guest and the host sides. But it has to be done with care. It should eliminate redundancy for the guest (some info reusable) and also make sure that the host always gets the info they absolutely need (like when arriving, how many nights requested, how many people).

**Comment by kalpaitch** [comment]
Just to highlight some member comments that we received today "Duplicate private messages impact responsiveness rating." - https://warmshowers.freshdesk.com/helpdesk/tickets/8592

There are still issues with the responsiveness rating being tied to Private Messages which would be resolved by a more formal hosting request process.
